templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Driver license</h1>

```

Comment:
The `yesno` filter is being used on `object`, which might not be a boolean value. Ensure that `object` is a boolean or adjust the logic to determine whether the form is for updating or creating a driver.